README.md

```diff
@@ -1,22 +1,25 @@
-# CerealNotes
+# Installation
+## Locally
+* postgres server installed and running. 
```

Comment:
nbd. we can clean this up in a later PR, but this line ends with a period, the next two don't, and the fourth one ends with a colon